### ISSUE TITLE
Why is the emcee import commented?

### ISSUE THREAD

**Issue by igomezv** [ja-vazquez/SimpleMC#34]
The emcee analyzer option is disabled:

https://github.com/ja-vazquez/SimpleMC/blob/6cb9d171ac907d5cb30d0cc4582eee02655b71bc/simplemc/DriverMC.py#L8

**Comment by seVenVo1d** [comment]
Will it be available in the future ?

**Comment by igomezv** [comment]
Yes, it is indeed available by uncommenting this line. However, we will notify you when it is tested.

**Comment by seVenVo1d** [comment]
I see, I was getting an error, and I have realized that there are two commented code lines. One in `simplemc/DriverMC` and the other one in `simplemc/analyzers/ __init__.py`. Now it works fine.

> However, we will notify you when it is tested.

That would be great, thanks.
